src/views/App.vue

```diff
@@ -176,6 +176,9 @@ export default {
   },
   created() {
     this.loadData();
+    setInterval(() => {
```

Comment:
Save the data returned by `setInterval` and call `clearInterval` on it when the App component is destroyed. (Like here: https://github.com/dangvanthanh/vue-clock/blob/master/src/VueClock.vue#L28)